tests/components/fibaro/test_config_flow.py

```diff
@@ -68,30 +28,28 @@ async def test_config_flow_user_initiated_success(hass: HomeAssistant) -> None:
     assert result["step_id"] == "user"
     assert result["errors"] == {}
 
-    with patch(
-        "homeassistant.components.fibaro.FibaroClient.connect",
-        return_value=True,
-    ):
-        result = await hass.config_entries.flow.async_configure(
-            result["flow_id"],
-            {
-                CONF_URL: TEST_URL,
-                CONF_USERNAME: TEST_USERNAME,
-                CONF_PASSWORD: TEST_PASSWORD,
-            },
-        )
-
-        assert result["type"] == "create_entry"
-        assert result["title"] == TEST_NAME
-        assert result["data"] == {
+    result = await hass.config_entries.flow.async_configure(
+        result["flow_id"],
+        {
             CONF_URL: TEST_URL,
             CONF_USERNAME: TEST_USERNAME,
             CONF_PASSWORD: TEST_PASSWORD,
-            CONF_IMPORT_PLUGINS: False,
-        }
+        },
+    )
 
+    assert result["type"] == "create_entry"
```

Comment:
```suggestion
    assert result["type"] == FlowResultType.CREATE_ENTRY
```
Change also the other tests

Reply:
Yes that makes sense, I changed it. Thanks for the hint.